src/components/Dashboard /PartnerData/index.js

```diff
@@ -11,26 +12,70 @@ import { toast } from "react-toastify";
 toast.configure();
 
 function PartnerDashboard() {
+  const [pageNumber, setPageNumber] = useState(0);
+  const [countPage, setCountPage] = useState(2);
+  const [isUpdateCountPage, setIsUpdateCountPage] = useState(true);
   const [partners, setPartners] = useState([]);
   const [searchTerm, setSearchTerm] = useState("");
   const [debouncedText] = useDebounce(searchTerm);
   const user = useSelector(({ User }) => User);
 
   useEffect(() => {
-    axios({
-      method: METHODS.GET,
-      url: `${process.env.REACT_APP_MERAKI_URL}/partners`,
-      headers: {
-        accept: "application/json",
-        Authorization: user.data.token,
-      },
-    }).then((res) => {
-      const orderedData = res.data.sort((a, b) => {
-        return b.users - a.users;
+    if (searchTerm.length > 0) {
+      axios({
+        method: METHODS.GET,
+        url: `${process.env.REACT_APP_MERAKI_URL}/partners?name=${searchTerm}`,
+        // url: `https://api.merakilearn.org/partners?name=${searchTerm}`,
+        headers: {
+          accept: "application/json",
+          Authorization: user.data.token,
+        },
+      }).then((res) => {
+        setPartners(res.data);
       });
-      setPartners(orderedData);
-    });
-  }, []);
+    } else {
+      axios({
+        method: METHODS.GET,
+        url: `${process.env.REACT_APP_MERAKI_URL}/partners?limit=${10}&page=${
+          pageNumber + 1
+        }`,
+        // url:`https://api.merakilearn.org/partners?limit=${10}&page=${pageNumber + 1}`,
+        headers: {
+          accept: "application/json",
+          Authorization: user.data.token,
+        },
+      }).then((res) => {
+        setPartners(res.data);
+        getCountPageNumber(res.data);
+      });
+    }
+  }, [searchTerm, pageNumber]);
+
+  const Partners = partners.filter((searchValue) => {
+    if (searchTerm == "") {
+      return searchValue;
+    } else if (
+      searchValue.name.toLowerCase().includes(searchTerm.toLowerCase())
+    ) {
+      return searchValue;
+    }
+  });
+
+  const getCountPageNumber = (response) => {
+    if (response.length === 0) {
+      setIsUpdateCountPage(false);
+    } else if (isUpdateCountPage) {
+      if (pageNumber > 10) {
+        setCountPage(pageNumber + 1);
+      } else {
+        setCountPage((preState) => preState + 1);
+      }
+    }
+  };
+
+  const changePage = ({ selected }) => {
+    setPageNumber(selected);
+  };
 
   const createMerakiLink = (id) => {
```

Comment:
change toast message to `Link created!` from `You have been created link successfully`